package.json

```diff
@@ -13,7 +13,7 @@
     "govuk-frontend": "^3.14.0"
   },
   "devDependencies": {
-    "@webpack-cli/serve": "^1.6.0",
+    "webpack-cli": "^4.9.0",
```

Comment:
The app already uses `webpack-cli` (see package.json#L39) to selectively resolve dependencies in webpack-dev-server@4.0.0

I would recommend bumping `webpack-cli` in the resolutions object instead of the adding this to the devDependencies object.